client/errors.go

```diff
@@ -0,0 +1,101 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package client
+
+// XXX: sync with the error kinds in daemon and ensure we define them
+//      only in a single place
+const (
+	// ErrorKindTwoFactorRequired the client needs to retry the login command including an OTP
+	ErrorKindTwoFactorRequired = "two-factor-required"
+	// ErrorKindTwoFactorFailed: the OTP provided wasn’t recognised
+	ErrorKindTwoFactorFailed = "two-factor-failed"
+	// ErrorKindLoginRequired: the requested operation cannot be performed without an authenticated user. This is the kind of any other 401 Unauthorized response.
```

Comment:
Will this be automatically processed and needs a single or can we wrap the comment?

Reply:
No automation yet, I have no strong opinion about wrapping or not wrapping.

Reply:
Unless there will be automation I think the line length should be reasonable.

Reply:
Updated